dss/stepfunctions/s3copyclient/implementation.py

```diff
@@ -141,6 +148,8 @@ def run(inner_self, subtask_id: int) -> None:
             assert all(results)
 
             state[Key.FINISHED] = True
+            if not cached and check_dss_bucket(self.destination_bucket):
```

Comment:
Since this check is repeated every time, should it be included in `_set_uncached_tag`?

Reply:
changed